classroom/snippets/snippets.py

```diff
@@ -60,3 +157,210 @@ def add_alias_existing(self):
         except errors.HttpError:
             print('Alias Creation Failed')
         # [END classroom_existing_alias]
+
+        def add_teacher(self):
```

Comment:
the spacing looks a little funky here, can you run pylint?

Reply:
ah yeah, i did + travis runs it as well -- i think it didn't catch it because decorators are a thing